scripts/microbe_stage/organelle.as

```diff
@@ -192,7 +192,19 @@ class Organelle{
         offset /= count;
         return offset;
     }
+    
+    Float3 calculateModelOffset() const
+    {
+        int count = 0;
 
+        auto keys = hexes.getKeys();
+        for(uint i = 0; i < keys.length(); ++i){
+            ++count;
+        }
+
+        return ((calculateCenterOffset()/=count))*HEX_SIZE;
```

Comment:
Using `hexes.getSize()` here would make for less duplicated code.

Reply:
did that